app/models/club_q.py

```diff
@@ -28,14 +28,24 @@ class ClubQSeason(db.Model):
     id: Column[int] = column(sa.Integer(), primary_key=True)
     nom: Column[str] = column(sa.String(64), nullable=False)
     promo_orga: Column[int] = column(sa.Integer(), nullable=False)
-    debut: Column[datetime.datetime | None] = column(sa.DateTime(), nullable=True)
-    fin: Column[datetime.datetime | None] = column(sa.DateTime(), nullable=True)
-    fin_inscription: Column[datetime.datetime | None] = column(sa.DateTime(), nullable=True)
+    debut: Column[datetime.date | None] = column(sa.Date(), nullable=True)
+    fin: Column[datetime.date | None] = column(sa.Date(), nullable=True)
+    fin_inscription: Column[datetime.date | None] = column(sa.Date(), nullable=True)
     spectacles: Relationship[list[ClubQSpectacle]] = one_to_many(
         "ClubQSpectacle.season", order_by="ClubQSpectacle.date"
     )
     voeu: Relationship[list[ClubQVoeu]] = one_to_many("ClubQVoeu.season", order_by="ClubQVoeu.id")
 
+    @property
+    def sum_places_demandees(self) -> int:
+        """The sum of all the places asked in a season"""
+        return sum(spect.sum_places_demandees for spect in ClubQSpectacle.query.filter_by(_season_id=self.id))
+
+    @property
+    def sum_places_attribuees(self) -> int:
+        """The sum of all the places asked in a season"""
+        return sum(spect.sum_places_attribuees for spect in ClubQSpectacle.query.filter_by(_season_id=self.id))
```

Comment:
Pourquoi ne pas utiliser la relation directement ?

```suggestion
        return sum(spect.sum_places_demandees for spect in self.spectacles)

    @property
    def sum_places_attribuees(self) -> int:
        """The sum of all the places asked in a season"""
        return sum(spect.sum_places_attribuees for spect in self.spectacles)
```

Reply:
C'est effectivement une riche idée